clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -2451,12 +2096,8 @@ Expected<SmallVector<StringRef>> linkAndWrapDeviceFiles(
       WrappedOutput.push_back(*OutputFile);
     }
     if (HasNonSYCLOffloadKinds) {
```

Comment:
If this `if` is supposed to be aligned to the upstream code, then I would suggest to add an explicit comment about it - it should help pulldown coordinators when dealing with merge conflicts

Reply:
I thought about this. I would prefer to track this more closely in the future than rely on co-ordinators to deal with merge conflicts.

Thanks